pages/collections/_type/_.vue

```diff
@@ -241,6 +241,14 @@
           && this.page.relatedLinksCollection.items.length > 0)
           ? this.page.relatedLinksCollection.items : null;
       },
+      relatedCollectionsFound() {
+        if (this.relatedEntities?.length > 0) {
+          return true;
+        } else if (this.relatedCollectionCards?.length > 0) {
+          return true;
+        }
```

Comment:
I guess you can even combine these. 
`if (this.relatedEntities?.length > 0 || this.relatedCollectionCards?.length > 0) {`